nilearn/regions/parcellations.py

```diff
@@ -194,48 +202,35 @@ class Parcellations(MultiPCA):
         masking.compute_background_mask, masking.compute_epi_mask or
         masking.compute_brain_mask. Default='epi'.
 
-    mask_args : dict, optional
+    mask_args : :obj:`dict`, optional
         If mask is None, these are additional parameters passed to
         masking.compute_background_mask or masking.compute_epi_mask
         to fine-tune mask computation. Please see the related documentation
         for details.
 
-    scaling : bool, optional
+    scaling : :obj:`bool`, optional
         Used only when the method selected is 'rena'. If scaling is True, each
         cluster is scaled by the square root of its size, preserving the
         l2-norm of the image. Default=False.
 
-    n_iter : int, optional
+    n_iter : :obj:`int`, optional
         Used only when the method selected is 'rena'. Number of iterations of
         the recursive neighbor agglomeration. Default=10.
-
-    memory : instance of joblib.Memory or str, optional
-        Used to cache the masking process.
-        By default, no caching is done. If a string is given, it is the
-        path to the caching directory.
-
-    memory_level : integer, optional
-        Rough estimator of the amount of memory used by caching. Higher value
-        means more memory for caching. Default=0.
-
-    n_jobs : integer, optional
-        The number of CPUs to use to do the computation. -1 means
-        'all CPUs', -2 'all CPUs but one', and so on.
-        Default=1.
-
-    verbose : integer, optional
-        Indicate the level of verbosity. By default, nothing is printed.
-        Default=0.
+    %(memory)s
+    %(memory_level)s
+    %(n_jobs)s
+    %(verbose0)s
```

Comment:
Sorry, but why `verbose0` instead of  `verbose` ?

Reply:
Because there are lots of verbose arguments defaulting to 0 (but most often to 1), I created two entries in the standard doc dict:

https://github.com/nilearn/nilearn/blob/65200aa3c81a7e70914ba85c4115c483f7510e8f/nilearn/_utils/docs.py#L26-L31

Instead of having the default added in each docstring.